### ISSUE TITLE
Handle how screen readers will interact with our application - Core interaction

### ISSUE THREAD

**Issue by duff92** [AxessLab/typing#9]
A few challenges have been noticed when using a screen reader (i.e. JAWS) in online applications. We need to solve how screen reader commands will interact with our application.

- Research `role="application"`
- `<div>` or `<input>`

Suggested output:
- Simple application showcasing basic interaction for typing commands

**Comment by AxessLab** [comment]
During a discussion with @DanielGoransson and Claudio, we discussed to explore `role='application'` within our application. According to a few resources, this is a way to tell the screen reader that our application should be running as a desktop application and therefore normal commands will not work. This can be useful when being in "game mode" to allow us to handle this interaction ourselves (keyboard input, sound feedback etc.) instead of using the screen reader.
